pkg/jx/cmd/create_jenkins_token.go

```diff
@@ -207,6 +207,7 @@ func (o *CreateJenkinsUserOptions) getAPITokenFromREST(serverURL string, userAut
 			return errors.Wrap(err, "parsing the timeout value")
 		}
 		ctx, cancel = context.WithTimeout(context.Background(), duration)
+		defer cancel()
```

Comment:
Not sure I understand why this was added. There was already a `defer cancel()` a few lines below.

(And the code being discussed was not touched in my PR. Did you think there was something wrong with it before?)

Reply:
right, I haven't seen that. It can be removed.